### ISSUE TITLE
sqlite: "database" path in ormconfig.json is not relative

### ISSUE THREAD

**Issue by stahlstift** [typeorm/typeorm#798]
### Given

```
# Example ormconfig.json
[
  {
    "name": "default",
    "type": "sqlite",
    "database": "./data/example.sq3",
    "autoSchemaSync": true,
    "entities": [
      "src/Model/*.js"
    ]
  }
]
```

```
#Filestructure
src/**.*
package.json
ormconfig.json
```

### My expection

```
data/example.sq3
src/**.*
package.json
ormconfig.json
```

### Result
Error: SQLITE_CANTOPEN: unable to open database file

### Question
Bug? Should I fix this and make a pull request?

**Comment by pleerock** [comment]
where it creates it right now?

**Comment by stahlstift** [comment]
I think in process.cwd()

**Comment by pleerock** [comment]
okay I think you can create a PR for this

**Comment by pleerock** [comment]
hey @stahlstift didn't you had a chance to work on it yet?

**Comment by stahlstift** [comment]
@pleerock No sorry. My laptop is broken... I think I can start next week on this topic.

**Comment by pleerock** [comment]
This change will be released in 0.1.2
